indigo_api/migrations/0032_search.py

```diff
@@ -43,7 +43,8 @@ class Migration(migrations.Migration):
         ),
         migrations.RunSQL(
             "CREATE INDEX indigo_api_document_search_vector_ix ON indigo_api_document USING gin(search_vector)",
-            "DROP INDEX indigo_api_document_search_vector_ix"
+            "DROP INDEX indigo_api_document_search_vector_ix",
+            elidable=True
```

Comment:
These migrations aren't elidable. They setup state for search in the DB which cannot be done any other way.

Reply:
I have partial code to split this out into a separate package, but it's not ready yet. I can take a look at it this afternoon.

Reply:
In master, this has all been pulled out of indigo (into indigo-search-psql) so, assuming we'll be merging master into this branch soon, it doesn't matter that we've removed these operations. So this is now fine.